ra/ra.go

```diff
@@ -677,19 +677,6 @@ func (ra *RegistrationAuthorityImpl) checkInvalidAuthorizationLimit(ctx context.
 // rate limit. This rate limit ensures a client can not create more than the
 // specified threshold of new orders within the specified time window.
 func (ra *RegistrationAuthorityImpl) checkNewOrdersPerAccountLimit(ctx context.Context, acctID int64, names []string, limit ratelimit.RateLimitPolicy) error {
```

Comment:
The tests are failing because this function no longer uses the input parameter `names`. That parameter should be removed from this function definition and from all call-sites.